client/src/components/ThankYou/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import * as ROUTES from "../../constants/routes";
+import { Link } from "react-router-dom";
+
+class ThankYou extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <div>
+        <div>
+          <h1>Thank You </h1>
```

Comment:
Can you change this to a `<h2>` tag and add a `!` to the end?

Reply:
I also think it should be "Thank you!" instead of "Thank You!"

Reply:
👍

Reply:
Done